plugins/net_plugin/net_plugin.cpp

```diff
@@ -3728,12 +3730,10 @@ namespace eosio {
             fc_wlog( logger, "bad block connection - ${cid}: #${n} ${id}...: unknown exception",
                      ("cid", cid)("n", ptr->block_num())("id", id.str().substr(8,16)));
          }
-         if( exception || !obh) {
-            if (!exception && !obh) {
-               if (prev_is_proper_svnn_block || !ptr->is_proper_svnn_block()) {
-                  fc_dlog(logger, "unlinkable_block ${bn} : ${id}, previous ${pn} : ${pid}",
-                          ("bn", ptr->block_num())("id", id)("pn", block_header::num_from_id(ptr->previous))("pid", ptr->previous));
-               }
+         if( exception || unlinkable) {
+            if (unlinkable && (prev_is_proper_svnn_block || !ptr->is_proper_svnn_block())) {
```

Comment:
Why this condition `&& (prev_is_proper_svnn_block || !ptr->is_proper_svnn_block())`?

Reply:
See comment lines 3722-3727 above.